package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "bin": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node_modules/.bin/mocha --opts spec/mocha.opts spec"
```

Comment:
you can just do `mocha --opts spec/mocha.opts spec` - node checks the `.bin` folder for you